pymodbus/framer/ascii_framer.py

```diff
@@ -32,12 +32,12 @@ class ModbusAsciiFramer(ModbusFramer):
 
     method = "ascii"
 
-    def __init__(self, decoder, client=None):
+    def __init__(self, decoder, **kwargs):
```

Comment:
kwargs indicates it takes any number of parameters, but it only takes client=, the change make it harder to understand the code.

Reply:
There was a misunderstanding on my site. After changing the code to allow missing peer information I forgot to revert the changes to the interface. Will correct that.